lib/fluent/plugin/out_buffered_stdout.rb

```diff
@@ -0,0 +1,38 @@
+#
+# Fluentd
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+require 'fluent/output'
+
+module Fluent
+  class BufferedStdoutOutput < BufferedOutput
```

Comment:
Thanks for the comment.
I think that `ObjectBufferedOutput` is not enough for this plugin because `ObjectBufferedOutput` does not implement format itself.
It seems to need to implement `format` in this plugin.